Controller/MediaAdminController.php

```diff
@@ -42,22 +41,20 @@ public function createAction(Request $request = null)
     /**
      * {@inheritdoc}
      */
-    public function render($view, array $parameters = array(), Response $response = null, Request $request = null)
+    public function render($view, array $parameters = array(), Response $response = null)
```

Comment:
It is, see my comment above, with a child class which has one more optional parameter (it's optional because it is after an optional parameter)